packages/code-du-travail-frontend/src/common/RelatedItems.js

```diff
@@ -28,6 +28,17 @@ export const RelatedItems = ({ items = [] }) => {
     .filter(({ source }) => !tileSources.includes(source))
     .slice(0, 6);
 
+  const matoSelectRelated = (reco, selection) => {
```

Comment:
slight performance improvement if you put this function outside the component (because the component will get reexecuted at each render)